ios/RNGestureHandler.m

```diff
@@ -390,14 +395,21 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
     if (_recognizer.state == UIGestureRecognizerStateBegan && _recognizer.state == UIGestureRecognizerStatePossible) {
         return YES;
     }
-    if ([_simultaneousHandlers count]) {
-        RNGestureHandler *handler = [RNGestureHandler findGestureHandlerByRecognizer:otherGestureRecognizer];
-        if (handler != nil) {
+    
+    RNGestureHandler *handler = [RNGestureHandler findGestureHandlerByRecognizer:otherGestureRecognizer];
+    if (handler != nil) {
+        if ([_simultaneousHandlers count]) {
             for (NSNumber *handlerTag in _simultaneousHandlers) {
                 if ([handler.tag isEqual:handlerTag]) {
                     return YES;
                 }
             }
+        } else if ([handler getSimultaneousHandlers]) {
```

Comment:
Ot just make `handler->_simultaniousHandlers` here

Reply:
Done in fe4b2ea.